master/buildbot/steps/trigger.py

```diff
@@ -44,12 +44,14 @@ class Trigger(BuildStep):
 
     flunkOnFailure = True
 
-    def __init__(self, schedulerNames=None, sourceStamp=None, sourceStamps=None,
+    def __init__(self, schedulerNames=None, unimportantSchedulerNames=None, sourceStamp=None, sourceStamps=None,
```

Comment:
my suggestion would be to add this new parameter before `**kwargs`: while all over the place we promote explicit parameter naming, some users might "rely" on a particular order of parameters.